docs/tutorials/wiki/src/authorization/tutorial/security.py

```diff
@@ -1,5 +1,16 @@
-USERS = {'editor':'editor',
-          'viewer':'viewer'}
+import bcrypt
+
+
+def hash_password(pw):
+    return bcrypt.hashpw(pw.encode('utf-8'), bcrypt.gensalt())
```

Comment:
I'm not strictly against it since zodb can just handle the bytes, and the data is ascii, but I think unicode is slightly more accurate.